source/configuration_manual/shared_mailboxes/cluster_setup_example.rst

```diff
@@ -0,0 +1,96 @@
+.. _mailbox_sharing_in_cluster_simple_example:
+
+===============================================================
+Example config snippets for shared mailboxes in Dovecot Cluster
+===============================================================
+
+The following snippets show important configuration parts for configuring
+cluster wide sharing of mailboxes. These are not complete configurations
+for more details please refer to :ref:`mailbox_sharing_in_cluster`
```

Comment:
I think it needs .. Maybe a period between these two sentences (... configuration. For more ...)?